### PR TITLE
service migration adaptations

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

**Comment by Herklos** [comment]
For a new version you have to update the README.md (the first line) and the CHANGELOG.md ;)

File: octobot_backtesting/api/exchange_data_collector.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+from octobot_backtesting.collectors.exchanges.exchange_collector import ExchangeDataCollector
+from octobot_commons.tentacles_management.advanced_manager import get_single_deepest_child_class
+
+
+async def collect_exchange_historical_data(config, exchange_name, symbols, time_frames=None):
```
Comment (Herklos) [inline octobot_backtesting/api/exchange_data_collector.py]: :+1:

File: octobot_backtesting/data/data_file_manager.py [hunk 2]
```diff
@@ -21,11 +21,11 @@
...
 
 from octobot_commons.symbol_util import merge_currencies
-from octobot_commons.time_frame_manager import TimeFrameManager
+from octobot_commons.time_frame_manager import find_min_time_frame
 
-from octobot_trading.constants import BACKTESTING_TIME_FRAMES_TO_DISPLAY, BACKTESTING_DATA_FILE_TIME_DISPLAY_FORMAT, \
+from octobot_backtesting.constants import BACKTESTING_TIME_FRAMES_TO_DISPLAY, BACKTESTING_DATA_FILE_TIME_DISPLAY_FORMAT, \
     BACKTESTING_DATA_FILE_TIME_READ_FORMAT, BACKTESTING_DATA_FILE_EXT, BACKTESTING_DATA_FILE_TIME_WRITE_FORMAT
-from octobot_trading.enums import BacktestingDataFormatKeys, BacktestingDataFormats
+from octobot_backtesting.enums import DataFormatKeys, DataFormats
```
Comment (Herklos) [inline octobot_backtesting/data/data_file_manager.py]: :+1: But it seems that enums.py of octobot_backtesting with the new DataFormatKeys and DataFormats is not modified in this PR.

Comment (GuillaumeDSM) [reply octobot_backtesting/data/data_file_manager.py]: right, it was now compiling from the beginning (BacktestingDataFormatKeys doesn't exist in octobot_trading)

**Pull request merged**